test/parallel/test_mxnet.py

```diff
@@ -52,6 +62,10 @@ def filter_supported_types(self, types):
            types = [t for t in types if t in ccl_supported_types]
         return types
 
+    def test_gpu_required(self):
```

Comment:
this method does not seem to be used anywhere, same for the same-name-methods in two other files

Reply:
The way these tests work is that if `HOROVOD_TEST_GPU` is set, then we require that the framework can use GPUs.  This is to workaround an issue where TensorFlow was compiling with GPU support, but unable to use it because the wrong version of CUDA was installed.